test/ruby_lsp_rails/document_symbol_test.rb

```diff
@@ -349,6 +349,20 @@ class FooModel < ApplicationRecord
         assert_equal("validates_with(Foo::BarClass)", response[0].children[1].name)
       end
 
+      test "correctly handles association callbacks with string and symbol argument types" do
+        response = generate_document_symbols_for_source(<<~RUBY)
+          class FooModel < ApplicationRecord
+            belongs_to :foo, "baz"
```

Comment:
I'm sure Rails won't let you list multiple associations this way, so let's put them on separate lines.